package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,347 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Principal Component Analysis (PCA) --- :mod:`MDAnalysis.analysis.pca`
```

Comment:
Ah! I waited to do this until diffusion maps got merged, I'll take care of
it right now.


On Fri, Aug 12, 2016 at 1:31 PM Max Linke <notifications@github.com> wrote:

> In package/MDAnalysis/analysis/pca.py
> <https://github.com/MDAnalysis/mdanalysis/pull/896#discussion_r74655939>:
>
> > +#
> > +# MDAnalysis --- http://www.MDAnalysis.org
> > +# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
> > +# and contributors (see AUTHORS for the full list)
> > +#
> > +# Released under the GNU Public Licence, v2 or any higher version
> > +#
> > +# Please cite your use of MDAnalysis in published work:
> > +#
> > +# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
> > +# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
> > +# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
> > +#
> > +
> > +"""
> > +Principal Component Analysis (PCA) --- :mod:`MDAnalysis.analysis.pca`
>
> Is this already included in the docs? I think @richardjgowers
> <https://github.com/richardjgowers> told you in the diffusion maps PR
> where you have to add something in the doc rests so that this module shows
> up as well.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/896/files/97c0e823b46b0f37d3f6fded59fced92dfa675ab#r74655939>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKcARligEEjiZnJyHncL-uAjVQbLEJ4Vks5qfNgmgaJpZM4JIXwX>
> .
>
-- 
Have a wonderful day,
John J. Detlefs